guillotina/api/search.py

```diff
@@ -3,12 +3,13 @@
 from guillotina.catalog.utils import reindex_in_future
 from guillotina.component import query_utility
 from guillotina.interfaces import ICatalogUtility
+from guillotina.interfaces import IContainer
 from guillotina.interfaces import IResource
 from guillotina.utils import get_content_path
 
 
 @configure.service(
-    context=IResource, method='GET', permission='guillotina.SearchContent', name='@search',
+    context=IContainer, method='GET', permission='guillotina.SearchContent', name='@search',
```

Comment:
Searching on a resource meaning that the resource is the basic path to search for.